cli/file_fetcher.rs

```diff
@@ -196,13 +197,17 @@ impl SourceFileFetcher {
   ///
   /// If `use_disk_cache` is true then remote files are fetched from disk cache.
   ///
-  /// If `no_remote_fetch` is true then if remote file is not found it disk
+  /// If `no_remote` is true then this method will fail for remote files.
+  ///
+  /// If `no_fetch` is true then if remote file is not found in disk
   /// cache this method will fail.
+
   fn get_source_file_async(
     self: &Self,
     module_url: &Url,
     use_disk_cache: bool,
-    no_remote_fetch: bool,
+    no_remote: bool,
+    no_fetch: bool,
```

Comment:
This argument and the functionality of `--no-fetch` is still required internally for this bit: https://github.com/denoland/deno/blob/07e78f64bbda01e6a66ee4b948e4ac5e0e86cb53/cli/file_fetcher.rs#L124-L129

Reply:
@nayeemrmn I don't understand why... Isn't it just renaming of the flag?

Reply:
Read #3384 for details of the difference in the flags. I'm saying here that the (no longer exposed) functionality of `--no-fetch` is still required for the above function, so we need both arguments here.

Reply:
This is a very confusing change.